### DISCUSSION TITLE
Something happend with search button / is not responsive

### DISCUSSION THREAD

**Discussion by johnfelipe** [community/community#60137] [New to GitHub]
https://github.com/community/community/assets/428820/4167d201-2c01-4a84-9ba0-b36d78bffe6f

**Comment by queenofcorgis** [comment]
Thanks for taking the time to record this video @johnfelipe - it looks like the video is cut off/unclear of what is happening would you mind elaborating further please?

**Comment by johnfelipe** [comment]
Im trying to report is search area is not responsive....
